x-pack/plugins/entity_manager/server/lib/entity_client.ts

```diff
@@ -117,9 +117,7 @@ export class EntityClient {
     });
 
     if (!definition) {
-      const message = `Unable to find entity definition [${id}]`;
-      this.options.logger.error(message);
-      throw new EntityDefinitionNotFound(message);
```

Comment:
@elastic/obs-entities Whoever calls this client should determine if the error is expected or if it should be logged. Otherwise, we will duplicate errors or even display error messages when we shouldn't.